internal/handlers/common/error.go

```diff
@@ -67,6 +67,10 @@ func NewErrorMsg(code ErrorCode, msg string) error {
 	return NewError(code, errors.New(msg))
 }
 
+func NewErrorMsgf(code ErrorCode, msg string, args []any) error {
```

Comment:
I specifically removed a similar function before: formatting a message should be caller's responsibility

Reply:
1. removed this func from
1. added an agreements in the Contributing file